go-in-memory-db.go

```diff
@@ -13,7 +14,13 @@ var addr = ":6380"
 func main() {
 
 	go log.Printf("started server at %s", addr)
-	err := redcon.ListenAndServe(addr,
+	cer, err := tls.LoadX509KeyPair("server.crt", "server.key")
```

Comment:
lets do a flag based init.
for example `./run --ssl=true --private-key=server.crt --public-key=server.keey`
other wise start it without ssl